### DISCUSSION TITLE
iNAV 3.0.2

### DISCUSSION THREAD

**Discussion by packetkiller** [iNavFlight/inav#7600] [Q&A]
Software: iNAV 3.0.2 on Windows10 using inav_3.0.2_MATEKH743.hex firmware.
Hardware: Lumenier LUX H7 HD and Mateksys M8Q-5883 compass not working. The compass works in betaflight 10.7.1 using betaflight betaflight_4.2.9_MATEKH743.hex firmware. I admit it could be iNAV configuration issue.  GPS works in iNAV and get telemetry in OSD, just no compass.

Pinout:
LUX H7 HD               M8Q-5883
-----------------------------------
G                              G
4V5                          5V
Tx2                           RX
Rx2                          TX
CL                            CL
DA                           DA

**Comment by DzikuVx** [comment]
Verify i2c_speed setting, as most probably the default value is not working with the wires and distance you use. Most probably BF has lower I2C speed
